build.gradle

```diff
@@ -16,13 +41,10 @@ java {
   }
 }
 
-repositories {
-    mavenCentral()
```

Comment:
This should probably stay. Also, could you please add `mavenLocal()` here, too. Makes it easier to test unreleased versions of your plugin in the future.
I ran into the following error without it.
```
Execution failed for task ':tiaTests'.
> Error while evaluating property 'agentJvmConfiguration' of task ':tiaTests'.
   > Could not resolve all dependencies for configuration ':teamscaleJaCoCoAgent'.
      > Could not find com.teamscale:teamscale-jacoco-agent:34.0.1-SNAPSHOT.
        Searched in the following locations:
          - https://repo.maven.apache.org/maven2/com/teamscale/teamscale-jacoco-agent/34.0.1-SNAPSHOT/maven-metadata.xml
          - https://repo.maven.apache.org/maven2/com/teamscale/teamscale-jacoco-agent/34.0.1-SNAPSHOT/teamscale-jacoco-agent-34.0.1-SNAPSHOT.pom
        Required by:
            project :

* Try:
> The project declares repositories, effectively ignoring the repositories you have declared in the settings.
   You can figure out how project repositories are declared by configuring your build to fail on project repositories.
   For more information, please refer to https://docs.gradle.org/8.8/userguide/declaring_repositories.html#sub:fail_build_on_project_repositories in the Gradle documentation.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.
```

Reply:
This is defined [here](https://github.com/cqse/tia-tutorial-gradle-junit5/blob/new_step7/settings.gradle#L15) instead. I did try this all locally with the 34.0.1-SNAPSHOT and there it worked 🤔

Reply:
Did you maybe add the `repositories` block manually when trying it out?

Reply:
Yeah, very possible. My bad then, sorry 🙈